doc/starterkit/k4MarlinWrapperCLIC/Readme.md

```diff
@@ -158,3 +158,26 @@ result=$?
 [ $result = "0" ] || [ $result = "4" ] && true
 ```
 
+### DD4hep Geometry Information
+
+The ``MarlinDD4hep::InitializeDD4hep`` processor can be replaced by the ``k4SimGeant4::GeoSvc`` and the
+``TrackingCellIDEncodingSvc`` the latter of which is part of the k4MarlinWrapper reposityr.
+
+For example:
+
+```python
+import os
+from Configurables import GeoSvc, TrackingCellIDEncodingSvc
```

Comment:
```suggestion
from Configurables import GeoSvc, TrackingCellIDEncodingSvc, INFO
```

Reply:
In many places there is a `from Configurables import *` but I prefer importing them like this so that your python tools can tell you that something is not defined, while this is less trivial to do with a `import *` (I get a warning when it doesn't know if it's coming from from an `import *`). Eventually I would like all the star imports to be removed...

Reply:
So you just read my mind?